src/internal/testutil/enterprise.go

```diff
@@ -14,10 +14,9 @@ import (
 func GetTestEnterpriseCode(t testing.TB) string {
 	acode, exists := os.LookupEnv("ENT_ACT_CODE")
 	if !exists {
-		t.Error("Enterprise Activation code not found in Env Vars")
+		t.Error("Enterprise activation code not found in environment variable (ENT_ACT_CODE)")
```

Comment:
<3

Reply:
Haha, guess who didn't have that environment variable set when he first tried to run a minikubetestenv test 😂